tests/utils/mqtt_broker.py

```
@@ -0,0 +1,194 @@
+import os
+import subprocess
+import logging.config
+import time
+import threading
+import sys
+
+sys.path.append(".")
+
+import external_server as _external_server
+from paho.mqtt.client import MQTTMessage as _MQTTMessage
+import paho.mqtt.subscribe as subscribe  # type: ignore
```

Ruff

`paho.mqtt.subscribe` imported but unused

Remove unused import: `paho.mqtt.subscribe`

(F401)